src/main/java/seedu/mark/logic/parser/ParserUtil.java

```diff
@@ -49,6 +50,21 @@ public static Name parseName(String name) throws ParseException {
         return new Name(trimmedName);
     }
 
+    /**
+     * Parses a {@code String folder} into a {@code Folder}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code folder} is invalid.
+     */
+    public static Folder parseFolder(String folder) throws ParseException {
```

Comment:
```suggestion
    public static Folder parseFolder(String folderName) throws ParseException {
```

Can we rename `folder` to `folderName`?